build.gradle

```diff
@@ -582,7 +582,7 @@ dependencies {
     implementation 'org.ldaptive:ldaptive:1.2.3'
     implementation 'com.nimbusds:nimbus-jose-jwt:9.39.1'
     implementation 'com.rfksystems:blake2b:2.0.0'
-
+    implementation 'com.password4j:password4j:1.8.2'
```

Comment:
should we remove bouncycastle dependency from here?

Reply:
Afaik BouncyCastle is still used in the Security plugin. It will be fully removed in the final stages of the FIPS work